### PR TITLE
Feature/levelloading

### PR DESCRIPTION
**Pull request by ALobker**
Level loading is now in a separate class

### CODE REVIEW

File: core/src/com/group66/game/helpers/LevelLoader.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            int numBalls = Config.NUM_BALLS_ROW;
+            int xoffset = 0;
+            if (i % 2 != 0) {
+                numBalls--; // one less on the odd rows
+                xoffset = Config.BALL_RAD;
+            }
+            for (int j = 0; j < numBalls; j++) {
+                int xpos = Config.BOUNCE_X_MIN + (2 * j + 1) * Config.BALL_RAD + xoffset;
+                int ballIndex = r.nextInt(Ball.MAX_COLORS + 1);
+                if (ballIndex != Ball.MAX_COLORS) { // max_colors is no ball
```
Comment (iiKoe) [inline core/src/com/group66/game/helpers/LevelLoader.java]: If I remember correctly we have from could have or something that specified we have more ball colors. To make this future prove, could you change the check for a No Ball to an other character (like '-' or whatever).

**Comment by jeroenoverman** [comment]
Maybe you can make the javadoc comments

File: core/src/com/group66/game/helpers/LevelLoader.java (hunk 1 above)
Comment (ALobker) [inline core/src/com/group66/game/helpers/LevelLoader.java]: No, the value is not passed into the function. It is merely a check if we have to skip this space.

**Comment by ALobker** [comment]
Added more comments

**Pull request merged**